### PR TITLE
Small documentation edits

### PR DESCRIPTION
**Pull request by HotelSierraWhiskey**
Corrected a few inconsistencies in README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,9 +106,9 @@ Don't forget the `.` at the end!
 
 ## Documentation
 
-Documentation along with a quick start guide can be found in the [docs/](/docs) directory.
+Documentation, along with the Quick Start Guide, can be found in the [docs/](/docs) directory.
```
Comment (wozeparrot) [inline README.md:109]: what's the reason for this change/capitalization?

Comment (HotelSierraWhiskey) [reply README.md:109]: From what I gathered, that is the current title of the specified document (albeit without the stylized "tinygrad").

**Comment by geohot** [comment]
All PRs must either fix bugs, improve speed (>20%), refactor on a conceptual level (big line reduction), or add tested functionality.
